doc/user-guide.md

```diff
@@ -551,6 +551,46 @@ func main() {
 * After adding new import paths to your operator project, run `go mod vendor` if a `vendor/` directory is present in the root of your project directory to fulfill these dependencies.
 * Your 3rd party resource needs to be added before add the controller in `"Setup all Controllers"`.
 
+#### Default Metrics exported with 3rd party resource 
+
+By default the projects built with SDK will be implemented to export metrics. For further information check the [metrics section][metrics_doc]. Then, in the `main.go` you will see:
+
+```go
+func serveCRMetrics(cfg *rest.Config) error {
+    
+    ...
+	filteredGVK, err := k8sutil.GetGVKsFromAddToScheme(apis.AddToScheme)
+	if err != nil {
+		return err
+	}
+
+    ...
+
+	// Generate and serve custom resource specific metrics.
+	err = kubemetrics.GenerateAndServeCRMetrics(cfg, ns, filteredGVK, metricsHost, operatorMetricsPort)
+	if err != nil {
+		return err
+	}
+``` 
+
+The `kubemetrics.GenerateAndServeCRMetrics` requires an RBAC permission to List all GroupVersionKinds in the List of watched namespaces, therefore, you might need to customize the implementation of the [k8sutil.GetGVKsFromAddToScheme](https://godoc.org/github.com/operator-framework/operator-sdk/pkg/k8sutil#GetGVKsFromAddToScheme) by implementing your own `GetGVKsFromAddToScheme` to avoid permission issues such as `Failed to list *unstructured.Unstructured`.
```

Comment:
But we are just adding here what is required for the user be able to implement their own method. Whiteout the explanation I do not think that they will be able to do that.

Reply:
Users should not have to re-implement a library function. If that is the case, we need to re-work the library function. What I'm suggesting with this change is they do some further filtering on the list returned instead of re-implementing the function. I'd like to see my change go in verbatim.

Reply:
Hi @estroz, 

I really love the idea of an RFE to improve the current solution. However, for now, users need to write their own method since it is customization for them. It was the way that this feature was designed, and you can check that indeed in the comments in the code. 

So, for now: This doc is in order to explain and clarifies when, why and how which. is the biggest current issue that has been impact us and users.  We need to doc this problem and make clear these points. Then, we can track an RFE for we improve the lib. However, it would not be part of this PR.


Please, let me know if it makes sense for you.

Reply:
I'm not suggesting we propose changes to a library function in this PR or at all. I'm sayin that what you're suggesting in this paragraph is not the right approach.